### DISCUSSION TITLE
How to convert radar data in to netcdf?

### DISCUSSION THREAD

**Discussion by htan2013** [Unidata/netcdf-java#1049] [Questions and Answers]
Dear all,

I was trying to convert Level-2 NEXRAD radar data into NetCDF (example attached). 
[KMKX20170313_000057_V06.zip](https://github.com/Unidata/netcdf-java/files/9221621/KMKX20170313_000057_V06.zip)

Link of the decoding NEXRAD by NetCDF Java: https://www.ncei.noaa.gov/products/radar/decoding-utilities-examples
However, I couldn't convert this data into NC file. After clicking "write file", nothing happened.
Can someone tell me the right way to convert radar data into NC file?

**Comment by lesserwhirls** [comment]
It looks like you are trying to use toolsUI. If so, the "write file" capabilities are currently broken. However, you can use a command line utility to achieve the same result. You will need a Java 11 runtime installed and a copy of netcdfAll.jar (available here: https://downloads.unidata.ucar.edu/netcdf-java/)

If you follow the instructions to run nccopy (https://docs.unidata.ucar.edu/netcdf-java/current/userguide/cdm_utility_programs.html#nccopy), you should be good to go.
